pyxrf/model/fileio.py

```diff
@@ -727,8 +727,9 @@ def save_data_to_txt(fpath, output_folder):
 
 
 def read_hdf_APS(working_directory,
-                 file_names, channel_num=3,
-                 spectrum_cut=3000):
+                 file_name, channel_num=3,
```

Comment:
is `channel_num` basically guaranteed to be 3 because of the detector setup at the XRF beamlines at the APS?